charts/opentelemetry-collector/values.schema.json

```diff
@@ -1498,6 +1498,10 @@
         "enabled": {
           "type": "boolean"
         },
+        "mode": {
+          "type": "string",
+          "enum": ["HPA", "CUSTOM"]
```

Comment:
Could these be lowercase? It's easier to type and it's not like they are constants in some programming language.